lib/config/validator.js

```diff
@@ -75,6 +77,14 @@ class Validator {
             logger.warning(`Passing "${this.webpackConfig.sharedCommonsEntryName}" to createSharedEntry() is not recommended, as it will override the built-in cache group by this name.`);
         }
     }
+
+    _validateCacheGroupNames() {
+        for (const groupName of Object.keys(this.webpackConfig.cacheGroups)) {
+            if (['defaultVendors', 'default'].includes(groupName)) {
```

Comment:
should we also include `vendors`, which is the name in webpack 4 ?

Reply:
should we also warn if you pass `this.webpackConfig.sharedCommonsEntryName` ?

Reply:
> should we also include vendors, which is the name in webpack 4 ?

Oh, so that's why we previously used `vendors` in `_validateSharedEntryName()`!
I couldn't find a reference to it in the doc anymore since the following page already got updated with `defaultVendors`: https://webpack.js.org/plugins/split-chunks-plugin/#optimizationsplitchunks

I think I'll add it and we'll remove it when we upgrade to Webpack 5.

> should we also warn if you pass `this.webpackConfig.sharedCommonsEntryName` ?

We could but I'm not sure it'll be really useful... people will probably either:

* start a new project and directly use `addCacheGroup()` since `createSharedEntry()` throws a deprecation warning
* see the deprecation warning and try to replace their `createSharedEntry()` calls by `addCacheGroup()`

Doesn't cost much to add a warning though.